models/shopify_graphql.yml

```diff
@@ -0,0 +1,1202 @@
+      - name: price_pres_currency_code
+        description: The three-letter code (ISO 4217 format) for the presentment currency used to display the total price.
+      - name: price_shop_amount
+        description: The total price of the line item when the order was created, in *shop* currency. This value doesn't include discounts.
+      - name: price_shop_currency_code
+        description: The three-letter code (ISO 4217 format) for the shop currency used to display the total price.
+      - name: product_id
+        description: The ID of the product that the line item belongs to. Can be null if the original product associated with the order is deleted at a later date.
+      - name: quantity
+        description: The number of items that were purchased.
+      - name: is_shipping_required
+        description: Whether the item requires shipping.
+      - name: sku
+        description: The item's SKU (stock keeping unit).
+      - name: is_taxable
+        description: Whether the item was taxable.
+      - name: title
+        description: The title of the product.
+      - name: total_discount_shop_amount
+        description: The total discount allocated to the line item in *shop* currency, including the total allocated to refunded and removed quantities. This value doesn't include order-level discounts.
+      - name: total_discount_pres_amount
+        description: The total discount allocated to the line item in *presentment* currency, including the total allocated to refunded and removed quantities. This value doesn't include order-level discounts.
+      - name: total_discount_shop_currency_code
+        description: The three-letter code (ISO å4217 format) for the shop currency used to display the total discount.
+      - name: total_discount_pres_currency_code
+        description: The three-letter code (ISO 4217 format) for the currency currency used to display the total discount.
+      - name: variant_id
+        description: The ID of the product variant.
+      - name: vendor
+        description: The name of the item's supplier.
+      - name: refunded_quantity
+        description: Quantity of the item that has been refunded.
+      - name: quantity_net_refunds
+        description: Quantity ordered, excluding refunds.
+      - name: variant_barcode
+        description: The barcode, UPC, or ISBN number for the product.
+      - name: variant_compare_at_price
+        description: The original price of the item before an adjustment or a sale in shop currency.
+      - name: variant_created_at
+        description: The date and time (ISO 8601 format) when the product variant was created.
+      - name: inventory_item_id
+        description: The unique identifier for the inventory item, which is used in the Inventory API to query for inventory information.
+      - name: media_id
+        description: The unique numeric identifier for a product variant's media. 
```

Comment:
This field isn't included in the end model. Is that expected?

Reply:
Should be included if `shopify_gql_using_product_variant_media` is set to True (it's false by default)